src/components/ActionsPanel/types.ts

```diff
@@ -0,0 +1,34 @@
+import type {ButtonProps} from '../Button';
+import type {DropdownMenuItem} from '../DropdownMenu';
+import type {QAProps} from '../types';
+
+export interface ActionItem {
```

Comment:
I suggest to rename this type to `ActionsPanelItem` or `ActionsPanelActionItem`. We try to name the types associated with a component from the name of that component. This helps to avoid collisions in the general index file and have more transparent type names.